axolotl-web/src/pages/About.vue

```diff
@@ -4,11 +4,14 @@
     <h1 class="title">Axolotl Beta {{config.Version}}</h1>
     <h2 class="subtitle" v-translate>A cross-plattform signal client</h2>
     <div class="description">
-      <span v-translate>This is a free and open source signal client written in golang and vuejs.
-      You can support the development of axolotl either by filling</span> <br/>
-      <a href="https://github.com/nanu-c/axolotl/issues" @click="openExtern($event, 'https://github.com/nanu-c/axolotl/issues')" ><span v-translate>issues at the bug tracker</span></a>. <br />
-      <span v-translate>or</span><br />
-      <span v-translate>by becoming a </span>
+      <translate>This is a free and open source signal client written in golang and vuejs.</translate>
+      <br />
+      <translate class="mr-1">
+        You can support the development of axolotl either by filling
+      </translate>
+      <a href="https://github.com/nanu-c/axolotl/issues" @click="openExtern($event, 'https://github.com/nanu-c/axolotl/issues')" v-translate >issues at the bug tracker</a>.
```

Comment:
Did you try to put the link inside the `<translate>` element and keep only one `translate` by english sentence? In the worst case, something like this is needed https://github.com/Polyconseil/vue-gettext/issues/48 but we should not have to go that far as we don't need interpolation here.